src/app/components/Sidebar/index.tsx

```diff
@@ -1,9 +1,91 @@
 import classNames from "classnames/bind";
+import { Link } from "react-router-dom";
+
+import HomeIcon from "@mui/icons-material/Home";
+import TimerIcon from "@mui/icons-material/Timer";
+import AssignmentIcon from "@mui/icons-material/Assignment";
+import FavoriteIcon from "@mui/icons-material/Favorite";
+import ArrowCircleLeftOutlinedIcon from "@mui/icons-material/ArrowCircleLeftOutlined";
+import ArrowCircleRightOutlinedIcon from "@mui/icons-material/ArrowCircleRightOutlined";
 
 import styles from "./Sidebar.module.scss";
+import { useState } from "react";
+
+// import { MenuItem } from "@mui/material";
 
 const cx = classNames.bind(styles);
 
+const menuList = [
+  {
+    path: "/",
+    name: "Trang chủ",
+    icon: <HomeIcon />,
+    isActive: "active",
+  },
+  {
+    path: "",
+    name: "Kiểm tra",
+    icon: <TimerIcon />,
+    isActive: "",
+  },
+  {
+    path: "",
+    name: "Tài liệu",
+    icon: <AssignmentIcon />,
+    isActive: "",
+  },
+  {
+    path: "",
+    name: "Yêu thích",
+    icon: <FavoriteIcon />,
+    isActive: "",
+  },
+];
+
 export const Sidebar = () => {
-  return <div className={cx("container")}>sidebar</div>;
+  const storageIsOpenState = Boolean(localStorage.getItem("isOpen"));
```

Comment:
you can see in utils -> storage.ts, create a function to do that.

Reply:
oki a